templates/openstack/compare.html

```diff
@@ -400,6 +421,9 @@ <h2>Contact us</h2>
   </div>
 </section>
 
+{% with first_item="_openstack_next_step", second_item="_cloud_newsletter", third_item="_openstack_further_reading" %}{% include "shared/contextual_footers/_contextual_footer.html" %}{% endwith %}
```

Comment:
I think you need a new first_item partial for this text... I couldn't find it anywhere.